src/IIIFPresentation/BackgroundHandler/BatchCompletion/BatchCompletionMessage.cs

```diff
@@ -1,11 +1,65 @@
-﻿namespace BackgroundHandler.BatchCompletion;
-
-public record BatchCompletionMessage(
-    int Id,
-    int CustomerId,
-    int Total,
-    int Success,
-    int Errors,
-    bool Superseded,
-    DateTime Started,
-    DateTime Finished);
+﻿using System.Text.Json.Serialization;
+
+namespace BackgroundHandler.BatchCompletion;
+
+public class BatchCompletionMessage
+{
+    [JsonRequired]
+    public int Id { get; set; }
+    
+    [JsonRequired]
+    public int Customer { get; set; }
+    
+    [JsonRequired]
+    public required int Count { get; set; }
+    
+    [JsonRequired]
+    public int Completed { get; set; }
+    
+    [JsonRequired]
+    public int Errors { get; set; }
+    
+    [JsonRequired]
+    public bool Superseded { get; set; }
+    
+    [JsonRequired]
+    public DateTime Submitted { get; set; }
+    
+    [JsonRequired]
+    public DateTime Finished { get; set; }
+}
+
+public class OldBatchCompletionMessage
```

Comment:
Comment outlining what makes it "old" and some details around when it can be deleted?